src/data.js

```diff
@@ -21,6 +21,64 @@
  * Keep this a commonjs export.
  */
 module.exports = [
+  {
+    name: 'Gurdeep Singh',
+    description: 'Full Stact Developer in PHP and ReactJS (MERN)',
+    url: 'https://www.gurdeep.net/uses',
```

Comment:
Two changes:
- Make sure your uses page is live
- Make sure the country is the flag emoji instead of IN